### PR TITLE
BUG: Package wrapper files generated by `ServerlessEnterprisePlugin`

### LINKED ISSUE #67: BUG: Serverless is no longer creating wrapper functions for serverless dashboard
Serverless dashboard creates wrapper functions for the handler methods which contains metadata used for logging lambda invocations.    When serverless-jetpack plugin is used these wrapper functions do not get created and the lambda functions end up failing because the files and functions no longer exist.  These functions are prefixed with `s_` like `s_my_function.js`

To enable dashboard on an existing serverless project add app and tenant:

```yml
  # serverless.yml
  app: my-app
  tenant: my-company
```

### PR DESCRIPTION
**Pull request by ryan-roemer**
* BUG: Hack a fix to generate wrapper files like `ServerlessEnterprisePlugin` does. Fixes #67 
* Infra: Add node13 to Travis CI matrix. Bump Appveyor to node10.
* Deps: Update to `serverless@^1.57.0` in all scenarios. Refactor local path plugins to use new syntax.

@alexdebrie -- I've got what I consider the "least bad" workaround of an awkward tension between SFE plugin generating assets last in line and then built-in packaging happening effectively first in line which means that a custom packager like Jetpack has no easy lifecycle hooks to get in and take over while still getting SFE generated files. If you or someone from the SFE team could review the comments and hack approach around the `patchesForEnterprise` function in `index.js` I'd super appreciate it! (Totally open to any other approach that would work, but this was unfortunately the best thing I could come up with and it's not great).

/cc @cbumstead

### CODE REVIEW

**Comment by alexdebrie** [comment]
@ryan-roemer Oof! Thanks for the heads up, and sorry we're making packaging difficult here. I'll check internally to see if anyone can take a look at this. Will get you an update one way or the other by mid-day tomorrow.

**Comment by medikoo** [comment]
@ryan-roemer if I understand correctly issue is that `before:package:createDeploymentArtifacts` hook as registered by Jetpack is invoked _before_ one registered by `@serverless/enterprise-plugin`. If it'll be other way round there won't be an issue. 

If that's the case, then I believe there's an easier workaround than one currently implemented in this PR (although still involves a hack):

Do not register for `'before:package:createDeploymentArtifacts'` in plugin constructor. Instead register for an`'initialize'` event (it's run unconditionally, right  _before_ other hooks are processed).

Then, in `'initialize'` listener, register callback for `'before:package:createDeploymentArtifacts'` (at this point `@serverless/enterprise-plugin`listener will already be there).

Of course, culprit is, that attaching a hook at this point requires a hacking approach as:

```javascript
const hooks = serverless.pluginManager.hooks;
if (!hooks['before:package:createDeploymentArtifacts']) {
  hooks['before:package:createDeploymentArtifacts'] = [];
}
hooks['before:package:createDeploymentArtifacts'].push({
  pluginName: 'Jetpack',
  hook: hookCallbackGuaranteedToBeRunAsLast
});
```

That's probably best you can do now, and  on our side, we'll think whether it's possible to introduce a non hacky way to that problem, at same time ensuring that `@serverless/enterpise-plugin` won't fail unexpectedly cause of some plugins.

**Comment by ryan-roemer** [comment]
@medikoo -- Looks like it worked! Here's current implementation if you want to re-review: https://github.com/FormidableLabs/serverless-jetpack/pull/68/files#diff-168726dbe96b3ce427e7fedce31bb0bcR121

File: index.js [hunk 1]
```diff
@@ -83,9 +83,58 @@ class Jetpack {
...
+      // like the Serverless Enterprise plugin run before us.
+      initialize: () => {
+        const { hooks } = serverless.pluginManager;
+        Object.keys(delayedHooks).forEach((event) => {
+          hooks[event] = (hooks[event] || []).concat({
+            pluginName: this.constructor.name,
+            hook: delayedHooks[event]
+          });
+        });
+      },
```
Comment (medikoo) [inline index.js]: Looks good 👍

Comment (ryan-roemer) [reply index.js]: Yay! Thanks for the review and suggestion!

**Comment by ryan-roemer** [comment]
Released in `serverless-jetpack@0.7.0`

**Pull request merged**